README.md

```diff
@@ -18,6 +18,15 @@ Bulma is constantly in development! Try it out now:
 ```sh
 npm install bulma
 ```
+
+### Import
+
+After NPM installation, import css to your project -
```

Comment:
After installation, you can import the CSS file into your project using this snippet: